docarray/array/storage/opensearch/find.py

```diff
@@ -166,7 +166,7 @@ def _find(
         :return: DocumentArray containing the closest documents to the query if it is a single query, otherwise a list of DocumentArrays containing
            the closest Document objects for each of the queries in `query`.
         """
-        query = np.array(query).astype(np.float)
+        query = np.array(query).astype(np.float32)
```

Comment:
why this?

Reply:
why touching opensearch at all?

Reply:
`numpy` has a new release three days ago and its version is updated to `1.24.0` in ci, where `np.float` and `np.int` is no longer available